client/src/test/java/com/spectralogic/tpfr/client/ReWrapStatusExpected.java

```diff
@@ -0,0 +1,33 @@
+package com.spectralogic.tpfr.client;
+
+import com.spectralogic.tpfr.client.model.Phase;
+
+public class ReWrapStatusExpected {
```

Comment:
You could implement this as a kotlin data class and it will implement getters automatically